airflow/providers/amazon/aws/operators/redshift_to_s3.py

```diff
@@ -101,13 +106,19 @@ def __init__(  # pylint: disable=too-many-arguments
             self.unload_options = list(self.unload_options) + ['HEADER', ]
 
     def execute(self, context):
+        if all([self.schema, self.table]):
+            select_query = "SELECT * FROM {schema}.{table}".format(schema=self.schema, table=self.table)
+        elif self.custom_select_query:
+            select_query = self.custom_select_query
+        else:
+            raise AirflowBadRequest(f"""Either (schema, table) or custom_select_query should be set.
+            They are ({self.schema},{self.table}) and {self.custom_select_query} now.""")
```

Comment:
```suggestion
            raise AirflowBadRequest("Either (schema, table) or custom_select_query should be set. "
                f"They are ({self.schema},{self.table}) and {self.custom_select_query} now.")
```
If you use triple quotes `"""` it will also print the spaces you have in front of `They`. And in this case you probably do not want that.